README.md

```diff
@@ -50,37 +39,65 @@ Note that these match the 2024.2 Desktop release.
+We recommend installing the latest version of Docker from these links.
+WebCSD will run on [Docker Desktop](https://docs.docker.com/engine/install/#desktop), however this is not recommended and may [require a license](https://www.docker.com/legal/docker-subscription-service-agreement/).
 
-1. Copy and rename the file `docker-compose.sample-On-Site-only-db-config.yml` to `docker-compose.db-config.yml`
-2. Edit the `volumes` section of that file to point to any in-house databases and edit the `environment` section to configure the application to recognise these databases.
+CHECK: Test your Docker installation as described at the above links. Also verify that you can log into the CCDC
+container registry.
 
-More information is given in the notes & example sections of the sample file. This acts as an [override file](https://docs.docker.com/compose/extends/) which you will have to include in the startup command.
+```sh
+docker login -u <user> -p <password> ccdcrepository.azurecr.io
 
-## CSD-Theory Web Database Configuration
+# or to be prompted for the password
+docker login -u <user> --password-stdin ccdcrepository.azurecr.io
+```
+
+### PostgreSQL Server Installation
 
-Instructions on setting up CSD-Theory Web can be found in the [wiki](https://github.com/ccdc-opensource/on-site-webcsd/wiki/Setting-up-CSD%E2%80%90Theory-Web)
+Once you have downloaded the postgres CSD database, please follow the instructions at
+[Setting Up Your PostgreSQL Server](https://github.com/ccdc-opensource/on-site-webcsd/wiki/Setting-up-your-PostgreSQL-server) to restore
+the database into your environment.
 
-## SSL Configuration
+CHECK: Can you connect to the database from Docker containers? These commands should run without errors.
+
+```sh
+# Pull postgres image
+docker pull postgres:14.13 
+# Run container
+docker run --name debug-postgres -e POSTGRES_PASSWORD=passwordhere -d postgres:14.13 
+# Shell into container
+docker exec -it debug-postgres sh 
+# Attempt to establish a connection to the database server
+psql -h database.server.name -p 5432 -U postgres -d csd_database
+```
+
+## Basic Configuration And Installation
+
+### SSL Configuration
 
 Connecting to your WebCSD server via https is now required.
 To configure this you will need an SSL certificate and private key.
 Ask your local IT staff to set these up for you.
 
-Please follow the instructions at [Configuring SSL](https://github.com/ccdc-opensource/on-site-webcsd/wiki/Configuring-SSL) to configure SSL.
-
+Please follow the instructions at [Configuring SSL](https://github.com/ccdc-opensource/on-site-webcsd/wiki/Configuring-SSL).
 If the WebCSD server is run without configuring SSL a built-in self-signed certificate will be used.
 This is not recommended as it is insecure.
 
-## Installation
+### WebCSD Test Installation
```

Comment:
Calling this a test installation makes it sound like it's separate to the actual installation. Could this be called WebCSD Basic Installation, or something similar, instead?

Reply:
Changed to "WebCSD Basic Installation".